contracts/protocols/mainnet/compound_iii/main.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+import "./interfaces.sol";
+import "./helpers.sol";
+
+/**
+ *@title Compund III Resolver
+ *@dev get user position, user configuration, market configuration.
+ */
+contract CompoundIIIResolver is CompoundIIIHelpers {
+    /**
+     *@dev get position of the user for all collaterals.
+     *@notice get position details of the user in a market including overall position data, collaterals, rewards etc.
+     *@param user Address of the user whose position details are needed.
+     *@param markets Array of addresses of the market for which the user's position details are needed
+     *@return userDatas Array of overall position details of the user including balances, nonce, rewards and flags.
+     *@return collateralDatas Array of datas related to the assets input for which user's collateral details are needed.
+     */
+    function getPositionAll(address user, address[] calldata markets)
+        public
+        returns (UserData[] memory userDatas, UserCollateralData[][] memory collateralDatas)
```

Comment:
Maybe create another struct with name `positionData` containing both `userDatas , UserCollateralData[]`, and return array of that struct?

Reply:
Done.